### DISCUSSION TITLE
Need help with Qwik.

### DISCUSSION THREAD

**Discussion by Dom-Cogan** [facebook/stylex#523] [Q&A] [answered]
### Describe the issue

Hello I am getting a compiler error when I’m developing my fresh blank Qwik project. It also produces the save error when I use ‘npm run Qwik build’ I still get that stylex.create should be compiled at build time with the babel-plugin. I’ve installed that plugin and even added a babel.config.js file copied right from the website. But that doesn’t fix it for build time or development. I’m lost I feel like I’m missing a step. The compile step specifically, but I’ve looked on YouTube videos and on the website and haven’t seen anything.

### Expected behavior

I expect to be able to write code then see the updates in my preview then see those changes on build. 

### Steps to reproduce

Npm create qwik@latest 
(Choose blank app) 
 Then install styleX
Then stylex babel plugin
Npm install
Npm run dev

**Comment by nmn** [comment]
Look at the [Qwik Starter Template](https://github.com/nmn/qwik-stylex)mentioned here (https://stylexjs.com/docs/learn/ecosystem/#starter-templates).

**Reply by Dom-Cogan** [reply]
This links back to the styleX page that the Qwik docs links back to here... I'm really not sure what or how you did this but telling me to look at the docs and then link me to the docs that link back to you, can I have more specific details? I've looked at the docs and your project.

**Reply by nmn** [reply, marked as answer]
I'm linking to this project, which is a starter project for Qwik:

https://github.com/nmn/qwik-stylex

I saw an issue created on that repo that something isn't working right now and @PatrickJS said he'll fix it. I'll ask him and get it updated so it's working again.

**Reply by Dom-Cogan** [reply]
oh ok i understand
